apps/aecore/src/aec_parent_connector.erl

```diff
@@ -453,7 +384,7 @@ get_pinning_data_(Mod, NetworkID) ->
         {ok, #{epoch             => PrevEpoch,
                height            => Height,
                block_hash        => BlockHash,
-               parent_payload    => Mod:encode_parent_pin_payload(#{epoch => PrevEpoch, height => Height, block_hash => BlockHash}),
+               parent_payload    => aeser_hc:encode_parent_pin_payload(#{epoch => PrevEpoch, height => Height, block_hash => BlockHash}),
```

Comment:
This assumes the same binary fits on each parent chain (since parent chain is not passed as an argument). I guess you checked that indeed this works for Doge and BTC

Reply:
I'm about to. Let's assume that. This refactoring is AE certified compatible only :)

Reply:
From early tests it looks pretty good tho